### PR TITLE
DROOLS-2373 : BRL 'is contained in the (comma separated) list' does not produce multiselect dropdown

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2373

https://github.com/kiegroup/drools/pull/2189
https://github.com/kiegroup/kie-wb-common/pull/2359
https://github.com/kiegroup/drools-wb/pull/1025

### CODE REVIEW

**Comment by Rikkola** [comment]
@jomarko here is the BRL fix.

File: drools-workbench-models/drools-workbench-models-datamodel-api/src/main/java/org/drools/workbench/models/datamodel/rule/InterpolationVariable.java [hunk 1]
```diff
@@ -21,55 +21,73 @@
...
-                                                                                                                           that.factField );
+        return equalOrNull(this.varName,
+                           that.varName)
+                && equalOrNull(this.dataType,
+                               that.dataType)
+                && equalOrNull(this.factType,
+                               that.factType)
+                && equalOrNull(this.factField,
+                               that.factField)
+                && equalOrNull(this.operator,
```
Comment (jomarko) [inline drools-workbench-models/drools-workbench-models-datamodel-api/src/main/java/org/drools/workbench/models/datamodel/rule/InterpolationVariable.java]: Thinking if we need this `equalOrNull`, it seems to me `Objects.equals` has same behavior.
```java
@Override
    public boolean equals(Object obj) {
        if (obj == null) {
            return false;
        }
        if (!(obj instanceof InterpolationVariable)) {
            return false;
        }
        InterpolationVariable that = (InterpolationVariable) obj;
        return Objects.equals(this.varName,
                              that.varName)
                && Objects.equals(this.dataType,
                                  that.dataType)
                && Objects.equals(this.factType,
                                  that.factType)
                && Objects.equals(this.factField,
                                  that.factField)
                && Objects.equals(this.operator,
                                  that.operator);
    }
```

**Review by jomarko — APPROVED** [review]
One comment but approving in advance.

**Review by manstis — APPROVED** [review]

**Comment by jomarko** [comment]
@Rikkola @manstis hi, this seems to me as fine to be merged. Is there anything that I am missing and block us from merging? Asking because we have to create 7.14.x cherry pick until Friday.

**Pull request merged**